lib/inquisition/rubocop/runner.rb

```diff
@@ -11,21 +11,39 @@ class Runner < ::Inquisition::Runner
         fatal: :high
       }.freeze
 
-      ::RuboCop::ConfigLoader::DOTFILE = 'config/.rubocop.yml'.freeze
+      USERS_CONFIG_FILE = '.rubocop.yml'.freeze
 
       def call
-        offenses = Inquisition::Rubocop::RuboCopModifiedRunner.new({}, ::RuboCop::ConfigStore.new).run(['.'])
+        offenses = RuboCopModifiedRunner.new(options, config_store).run(['.'])
         offenses.each { |offense| create_issue(offense) }
         @issues
       end
 
       private
 
-      def create_issue(offense)
-        offense_body = offense.last.first
-        @issues << Inquisition::Issue.new(severity: LEVELS[offense_body.severity.name],
-                                          path: offense.first, message: offense_body.message,
-                                          runner: self, line: offense_body.line)
+      def options
+        { config: choose_config }
+      end
+
+      def config_store
+        config_store = ::RuboCop::ConfigStore.new
+        config_store.options_config = options[:config]
+        config_store
+      end
+
+      def choose_config
+        return USERS_CONFIG_FILE if File.exist?(USERS_CONFIG_FILE)
+
+        gem_root = File.realpath(File.join(File.dirname(__FILE__), '..', '..'))
```

Comment:
https://stackoverflow.com/questions/10132949/finding-the-gem-root

Propose to move this method to `Inquisition.root` + spec